docs/samples/Microsoft.ML.Samples/Dynamic/ConvertType.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class ConvertTypeTransform
+    {
+        private sealed class InputData
+        {
+            public bool Survived;
+        }
+
+        private sealed class TransformedData
+        {
+            public bool Survived { get; set; }
+
+            public Int32 SurvivedInt32 { get; set; }
+        }
+
+        public static void Example()
```

Comment:
>Example [](start = 27, length = 7)

the sample needs to be linked from the actual source it illustrates. 

Checkout the <example> node on the other extension methods.

Reply:
fixed

---
In reply to: [261047048](https://github.com/dotnet/machinelearning/pull/2781#discussion_r261047048) [](ancestors = 261047048)